packages/frontend-acceptance-tests/src/pages/page.js

```diff
@@ -20,6 +20,9 @@ class Page {
       if (url.includes('?')) {
         url = url.substr(0, url.indexOf('?'))
       }
+      if (url.includes('#')) {
+        url = url.substr(0, url.indexOf('#'))
+      }
```

Comment:
Rather than this, I think it'd be neater to do
```
const charactersToIgnore = ['?', '#']
for (const characterToIgnore of charactersToIgnore) {
  if (url.includes(characterToIgnore)) {
    url = url.substr(0, url.indexOf(chacterToIgnore))
  }
}
```